### PR TITLE
Qu auto/sorqa 329

### PR DESCRIPTION
**Pull request by lb-sgent**
image

### CODE REVIEW

**Comment by lb-sgent** [comment]
@rdutu-vg please review.

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/dashboard/contacts/ContactsDashboardSteps.java [hunk 1]
```diff
@@ -57,5 +64,35 @@ public ContactsDashboardSteps(WebDriverHelpers webDriverHelpers) {
...
+
+    Then(
+        "I get Confirmed Contact labels and value from Contact Dashboard with English language",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(
+              CONFIRMED_COUNTER_LABEL_ON_CONTACTS_DASHBOARD);
+          webDriverHelpers.getWebElement(CONFIRMED_COUNTER_LABEL_ON_CONTACTS_DASHBOARD);
+          confirmedContact_EN =
+              webDriverHelpers.getWebElement(CONFIRMED_COUNTER_ON_CONTACTS_DASHBOARD).getText();
+          // System.out.println(confirmedContact);
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/dashboard/contacts/ContactsDashboardSteps.java:76]: remove commented

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/dashboard/contacts/ContactsDashboardSteps.java [hunk 2]
```diff
@@ -57,5 +64,35 @@ public ContactsDashboardSteps(WebDriverHelpers webDriverHelpers) {
...
+        });
+    Then(
+        "I get Confirmed Contact labels and value from Contact Dashboard with Deutsch language",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(
+              CONFIRMED_COUNTER_LABEL_ON_CONTACTS_DASHBOARD_DE);
+          webDriverHelpers.getWebElement(CONFIRMED_COUNTER_LABEL_ON_CONTACTS_DASHBOARD_DE);
+          confirmedContact_DE =
+              webDriverHelpers.getWebElement(CONFIRMED_COUNTER_ON_CONTACTS_DASHBOARD_DE).getText();
+          // System.out.println(confirmedContact);
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/dashboard/contacts/ContactsDashboardSteps.java:88]: remove commented

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/dashboard/surveillance/SurveillanceDashboardSteps.java [hunk 3]
```diff
@@ -1032,5 +1040,35 @@ public SurveillanceDashboardSteps(WebDriverHelpers webDriverHelpers, SoftAssert
...
+              webDriverHelpers
+                  .getWebElement(CONFIRMED_COUNTER_ON_SURVEILLANCE_DASHBOARD_DE)
+                  .getText();
+
+          webDriverHelpers.getWebElement(CONFIRMED_CURVE_ON_SURVEILLANCE_DASHBOARD);
+        });
+    And(
+        "I compare English and German confirmed counter",
+        () -> {
+          Assert.assertEquals(confirmedCases_EN, confirmedCases_DE, "Counters not equal!");
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/dashboard/surveillance/SurveillanceDashboardSteps.java:1071]: be more specific in assert message regarding which counter

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/dashboard/contacts/ContactsDashboardSteps.java [hunk 4]
```diff
@@ -57,5 +64,35 @@ public ContactsDashboardSteps(WebDriverHelpers webDriverHelpers) {
...
+
+          webDriverHelpers.getWebElement(UNDER_FU_CHART_ON_CONTACTS_DASHBOARD);
+        });
+    And(
+        "I compare English and German confirmed contacts counter",
+        () -> {
+          Assert.assertEquals(confirmedContact_EN, confirmedContact_DE, "Counters not equal!");
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/dashboard/contacts/ContactsDashboardSteps.java:95]: be more specific in assert message regarding which counter

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

**Comment by lb-sgent** [comment]
image
@rdutu-vg fixed. Please check again.

**Pull request merged**